### PR TITLE
PLG-107: Extract image url instead of GUID

### PR DESCRIPTION
**Pull request by nedick007**
Simple modification that changes the way we extract image url's by replacing current logic of using GUID (witch can change in some scenarios like domain change and result in wrong url).

### CODE REVIEW

**Comment by nedick007** [comment]
@adamof ready for review

File: includes/integration.php [hunk 1]
```diff
@@ -375,8 +375,9 @@ public function sync_orders_chunk($specific_order_ids = false){
 
                             // fetch image URL
                             $image_id = get_post_thumbnail_id($product['product_id']);
-                            $image = get_post($image_id);
-                            if($image && $image->guid) $product_hash['image_url'] = $image->guid;
+                            
+                            $image_url = wp_get_attachment_image_src($image_id, 'full')[0];
```
Comment (adamof) [inline includes/integration.php]: can you somehow check the availability of this function in old WP/WC versions?

Comment (nedick007) [reply includes/integration.php]: Compatible back to wordpress 2.5.0 (year 2008).

**Comment by nedick007** [comment]
@adamof updated product event as well, found no more uses for img_url except order import and product event.

File: includes/integration.php [hunk 2]
```diff
@@ -577,8 +578,8 @@ public function prepare_product_hash($product, $variation_id = false, $variation
         }
         // fetch image URL
         $image_id = get_post_thumbnail_id($product_id);
-        $image = get_post($image_id);
-        if($image && $image->guid) $product_hash['image_url'] = $image->guid;
+        $image_url = wp_get_attachment_image_src($image_id, 'full')[0];
+        if($image_url) $product_hash['image_url'] = $image_url;
```
Comment (adamof) [inline includes/integration.php:585]: Instead of copying and pasting the code around why not make a shared function?

Comment (nedick007) [reply includes/integration.php:585]: New function for img url is added.

**Comment by nedick007** [comment]
@adamof bump

**Comment by nedick007** [comment]
@adamof ready

**Review by adamof — APPROVED** [review]

**Comment by nedick007** [comment]
@adamof tested on QA env.

**Comment by adamof** [comment]
ok, merge and release a new version :)

**Pull request merged**